### PR TITLE
Script for finding missing tooltips

### PR DESCRIPTION
**Pull request by chrisinajar**

### CODE REVIEW

File: game/resource/English/ability/items/tooltip_reactive_2b.txt [hunk 1]
```diff
@@ -8,3 +8,6 @@
             "DOTA_Tooltip_Ability_item_reactive_2b_bonus_health"                                "+$health"
             "DOTA_Tooltip_Ability_item_reactive_2b_bonus_armor"                                 "+$armor"
             "DOTA_Tooltip_Ability_item_reactive_2b_magic_resistance"                            "%+$spell_resist"
+
+            "dota_tooltip_modifier_item_reactive_2b"                                            "Spell Block"
+            "dota_tooltip_modifier_item_reactive_2b_description"                                "Hit me, I date you"
```
Comment (Trildar) [inline game/resource/English/ability/items/tooltip_reactive_2b.txt]: "I date you" XD

File: game/resource/English/ability/items/tooltip_shivas_cuirass.txt [hunk 2]
```diff
@@ -37,3 +37,4 @@
             "DOTA_Tooltip_Ability_item_shivas_cuirass_2_blast_debuff_duration"              "BLAST SLOW DURATION:"
             "DOTA_Tooltip_Ability_item_shivas_cuirass_2_blast_radius"                       "BLAST RADIUS:"
 
+            "dota_tooltip_modifier_item_shivas_cuirass_negative"                            "Shiva's Curse"
```
Comment (Trildar) [inline game/resource/English/ability/items/tooltip_shivas_cuirass.txt]: IIRC from looking at the Shiva's Cuirass code, the positive and negative aren't actually used. Probably set it up, changed to a single modifier cause it was simpler to implement then forgot to remove them.

File: game/scripts/npc/items/item_travel_boots_2.txt [hunk 3]
```diff
@@ -35,7 +35,7 @@
 		// General
 		//-------------------------------------------------------------------------------------------------------------
 		"ID"							"220"														// unique ID number for this item.  Do not change this once established or it will invalidate collected stats.
-		"BaseClass"                     "item_travel_boots"
+		"BaseClass"                     "item_travel_boots_2"
```
Comment (chrisinajar) [inline game/scripts/npc/items/item_travel_boots_2.txt]: I have no idea if this works, i didn't test it.

**Comment by chrisinajar** [comment]
I mostly just want this test case to run for future merges, we'll go fast and loose with it and fix it if it's broken later.

**Pull request merged**